actions/earned-income-budgeting.vbs

```diff
@@ -4030,64 +4939,119 @@ If update_with_verifs = TRUE Then       'this means we have at least one panel w
+										If IsDate(this_date) = TRUE Then
+											date_found = FALSE      'default for the start of each loop
+											checks_found = 0
+											combined_gross_pay = 0
+											For all_income = 0 to UBound(LIST_OF_INCOME_ARRAY, 2)   'then loop through all of the income information
+												'conditional if it is the right panel AND the order matches - then do the thing you need to do
+												If LIST_OF_INCOME_ARRAY(panel_indct, all_income) = ei_panel AND LIST_OF_INCOME_ARRAY(pay_date, all_income) <> "" Then
+													If DateDiff("d", LIST_OF_INCOME_ARRAY(pay_date, all_income), this_date) = 0 Then        'if the date was in the array, we use it
+														date_found = TRUE
+														checks_found = checks_found + 1
+														CASH_MONTHS_ARRAY(retro_updtd, next_cash_month) = TRUE      'setting this to know there was retro information added
+														Call create_mainframe_friendly_date(LIST_OF_INCOME_ARRAY(view_pay_date, all_income), jobs_row, 25, "YY")		'pay date
+														combined_gross_pay = combined_gross_pay + LIST_OF_INCOME_ARRAY(gross_amount, all_income)
+														total_hours = total_hours + LIST_OF_INCOME_ARRAY(hours, all_income)             'running total of hours
+														total_pay = total_pay + LIST_OF_INCOME_ARRAY(gross_amount, all_income)          'running total of pay for RETRO month only
+														count_checks = count_checks + 1                 'need to track the number of checks that we used
+														updates_to_display = updates_to_display & vbNewLine & "Date - " & LIST_OF_INCOME_ARRAY(view_pay_date, all_income) & " - $" & LIST_OF_INCOME_ARRAY(gross_amount, all_income)
+														income_items_used = income_items_used & all_income & "~"
+													End If
+												End If
+											Next
+											If date_found = TRUE Then
+												combined_gross_pay = FormatNumber(combined_gross_pay, 2, -1, 0, 0)
+												EMWriteScreen combined_gross_pay, jobs_row, 38              'gross pay - not using the excluded amount on the main panel
+												If checks_found > 1 Then
+													For all_income = 0 to UBound(LIST_OF_INCOME_ARRAY, 2)   'then loop through all of the income information
+														If LIST_OF_INCOME_ARRAY(panel_indct, all_income) = ei_panel AND LIST_OF_INCOME_ARRAY(pay_date, all_income) <> "" Then
+															If DateDiff("d", LIST_OF_INCOME_ARRAY(pay_date, all_income), this_date) = 0 Then LIST_OF_INCOME_ARRAY(combined_into_one, all_income) = True
+														End If
+													Next
+												End If
+												jobs_row = jobs_row + 1         'moving to the next row
+											End If
+											'there is no functionality for if pay was not found as we don't use averages on the retro side
+										Else
+											testing_run = TRUE
+											script_run_lowdown = script_run_lowdown & vbCR & vbCR & "******* THIS_DATE IS NOT A DATE ********"
```

Comment:
5026: Did you mean to use "_" instead of a space between THIS DATE?

Reply:
Yes, this is referencing the variable. This output is only ever seen by script writers as it is part of the error reporting. I've added a comment to clarify what this is for.